nsv13/code/game/objects/items/nsv_circuitboards.dm

```diff
@@ -145,16 +165,32 @@
 #undef PATH_FLAK
 
 //Deck Gun
+/obj/item/circuitboard/deck_turret
```

Comment:
this should be refactored to ```/obj/item/circuitboard/machine/deck_turret``` so that it can actually be placed in machine frames